lms/static/js/bookmarks/views/bookmarks_list.js

```diff
@@ -59,8 +59,24 @@
                 });
             },
 
-            visitBookmark: function (event) {
-                window.location = event.target.pathname;
+            visitBookmark: function (url) {
+                window.location.href = url;
+            },
+
+            logBookmarkedItem: function (event) {
```

Comment:
Just `logBookmarkedItem` is a bit misleading. I would just call this `visitBookmark` and bring that one line inside `always()`.